### ISSUE TITLE
Reproducing with own data

### ISSUE THREAD

**Issue by brenopoggiali** [luis-gonzales/pointnet_own#3]
Hey, Luis! How are you doing?

Thank you so much for this repository! It is a great application of PointNet using TF2. 

I could already reproduce the result using ModelNet40 data. Now, I'm trying to reproduce the results using my collection of point clouds data, but I'm facing some issues.

Besides changing the dictionaries at ``dataset-utils.py`` and ``inference.py``, there are any other files that I need to change?

I'm also having trouble trying to open/edit the ``iter-51042`` files. I feel that there is something inside it that I should change, but when I open these files, I'm not able to read what's inside it. Do you open it with some specific program?

Thank you again!
Breno

**Comment by Tojens** [comment]
Can't help sadly, but if you figure anything out, I'd be interested in hearing from you

**Comment by luis-gonzales** [comment]
Apologies for the delay -- things got a bit crazy on my end.

For the most part, the changes required should be confined to `dataset_utils.py`. Keep in mind that your point clouds should be normalized to the unit sphere for object classification.

You're probably better off consulting the TF documentation as to the structure of the new tf saving format (see execution below). I'm not sure to what extent you should be modifying the raw file. The weights and biases probably aren't human readable.

`model.save_weights('filename', save_format='tf')`

**Comment by brenopoggiali** [comment]
Hey, @luis-gonzales! 

I solved the problem. Thank you for your answer. I really needed to transform my data to the unit sphere. Actually, if you want, I can open a pull request adding this to your ``pcd_to_npy.py`` script.

I also discovered that those ``iter-XXXX`` files are just the files containing the model trained. When evaluating, we should pass a parameter using the ``models/checkpoint/XXXXXXXX_XXXX/iter-XXXXX``, where it is the last iteration of the training that you want.

@Tojens , I'm not sure if this is going to help you, but I'll be glad to help you in any other struggling that you have.

Cheers!
